MetaMask/metamask-mobile
# Token Detection not persisting

**Description**
Tokens detected and added to the wallet were not persisting after we kill and re-launch the app

**Proposed Solution**
Working as expected, the tokens are persisting by the network.

**Test Cases**
Case1:
* Import an account with tokens
* Imported the tokens on mainnet
* kill the app
* re-launch the app
* The tokens persisted
Case2:
* Add avalanche network
* Add tokens
* kill the app
* relaunch the app
* should open on avalanche network with tokens persisted


**Screenshots/Recordings**
https://recordit.co/m3XogMlJw8


**Issue**

Progresses #5576 I truly believe this one it's already been fixed since doesn't happen to me on this PR

**Checklist**
